.prettierignore

```diff
@@ -0,0 +1,12 @@
+elixir/_build
+elixir/deps
+elixir/apps/web/assets
+rust/target
```

Comment:
```suggestion
rust
```

We have our own formatter :)

Reply:
Hmmm but there are typescript, markdown and other files in here. Maybe I should just exclude `*.rs` and `*.toml`?

Reply:
prettier ignores rust code anyway - it's not supported natively.